api4/system.go

```diff
@@ -64,11 +65,32 @@ func getSystemPing(c *Context, w http.ResponseWriter, r *http.Request) {
 	if r.FormValue("get_server_status") != "" {
 		dbStatusKey := "database_status"
 		s[dbStatusKey] = model.STATUS_OK
-		_, appErr := c.App.Srv.Store.System().Get()
-		if appErr != nil {
-			mlog.Debug(fmt.Sprintf("Unable to get database status: %s", appErr.Error()))
+
+		// Database Write/Read Check
+		currentTime := fmt.Sprintf("%d", time.Now().Unix())
+		healthCheckKey := "health_check"
+
+		writeErr := c.App.Srv.Store.System().SaveOrUpdate(&model.System{
+			Name:  healthCheckKey,
+			Value: currentTime,
+		})
+		if writeErr != nil {
+			mlog.Debug(fmt.Sprintf("unable to write to database: %s", writeErr.Error()))
```

Comment:
nit: It looks like our other ping debug logs start with capital letters so doing the same here would be ideal.

Reply:
True, I will update all the logging to be uppercase.